docs/azure_kms.md

```diff
@@ -1,5 +1,404 @@
 ---
 id: azure_kms
-title: Azure KMS
+title: AzureKMS
 ---
-Azure setup and operational docs will go here
\ No newline at end of file
+# Azure Key Vault
+
+The goal of this guide is to configure Signatory to use an Azure Key Vault as a signing backend.
+
+To setup Azure Key Vault as a signing backend for Signatory, you will need:
+
+* An active Azure subscription
+* The [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest) installed and configured on your computer.
+
+## **Azure setup**
+
+You will need to create several Azure resources, and copy configuration into the signatory config file. Let's begin.
+
+This guide uses the `az` cli command to create all required resources. For each command, you will get a json formatted result, or an error.
+
+### **Log in**
+
+```sh
+az login
+```
+
+### **Create an Azure resource group.**
+
+You need to specify the location you want your Signatory to be located. This is up to you. The closer to your baker the better, but other criteria may be important to you.
+
+```sh
+az group create \
+    --name RESOURCE_GROUP \
+    --location LOCATION
+```
+
+Example:
+
+```sh
+az group create --name "signatory" --location "canadaeast"
+```
+
+### **Create a new Key Vault, with HSM enabled**
+
+Next we will create a new Key Vault within our newly created resource group.
+
+```sh
+az keyvault create \
+    --name KEYVAULT_NAME \
+    --resource-group RESOURCE_GROUP \
+    --sku "premium"
+```
+
+Example:
+
+```sh
+az keyvault create --name "sigy" --resource-group "sigy" --sku "premium"
+```
+
+The `--sku` argument must be set to premium if you want to have your keys stored in a HSM.
```

Comment:
I think this is something to put in an urgent section with something like :caution: or something like that. If people don't get this right then things wont behave like they expect right?